client/jsconfig.json

```diff
@@ -3,7 +3,7 @@
     "paths": {
       "@assets/*": ["./src/assets/*"],
       "@components/*":["./src/components/*"],
-      "@General/*": ["src/components/core/*"],
+      "@General/*": ["./src/components/core/*"],
```

Comment:
👏

Reply:
It came that way, so I'm not sure! I just added the ./ before src since I was getting an error.

Reply:
I think for the sake of consistency we should update it to be `general`. The other aliases are not using caps anyways :)